### PR TITLE
EM-2243: Create ITHC charts for Evidence Management Applications

### PR DESCRIPTION
**Pull request by tomxelliott**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/ithc/common/dg/dg-docassembly.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/dg/docassembly:prod-461fa3fc
+      environment:
+        DOCMOSIS_TEMPLATES_ENDPOINT: "https://docmosis.platform.hmcts.net"
+        DOCMOSIS_ENDPOINT: "https://docmosis.platform.hmcts.net/rs/render"
+        S2S_BASE_URI: "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal"
+        DM_STORE_APP_URL: "http://dm-store-prod.service.core-compute-prod.internal"
+        ENABLE_FORM_DEFINITION_ENDPOINT: "false"
+        ENABLE_TEMPLATE_RENDITION_ENDPOINT: "true"
+        IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
```
Comment (luigibk) [inline k8s/ithc/common/dg/dg-docassembly.yaml]: Are you sure all of the above should be pointing to prod?

File: k8s/ithc/common/dg/identity.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: dg
+  namespace: dg
+spec:
+  type: 0
+  ResourceID: "/subscriptions/8cbc6f36-7c56-4963-9d36-739db5d00b27/resourceGroups/managed-identities-prod-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/dg-prod-mi"
+  ClientID: "038cdd7b-bd29-47c4-a2c5-923980222cc7"
+
```
Comment (luigibk) [inline k8s/ithc/common/dg/identity.yaml]: Prod again?

File: k8s/ithc/common/em/em-anno.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    anno:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/anno:prod-d8ae6a32
+      ingressHost: em-anno-prod.service.core-compute-prod.internal
+      environment:
+        S2S_BASE_URI: "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal"
+        IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
+    global:
```
Comment (luigibk) [inline k8s/ithc/common/em/em-anno.yaml]: As as above

File: k8s/ithc/common/em/em-ccd-orchestrator.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/ccd-orchestrator:prod-641e9a19
+      ingressHost: em-ccd-orchestrator-prod.service.core-compute-prod.internal
+      environment:
+        IDAM_API_BASE_URI: https://idam-api.platform.hmcts.net
+        S2S_BASE_URI: http://rpe-service-auth-provider-prod.service.core-compute-prod.internal
+        EM_STITCHING_API_URL: http://em-stitching-prod.service.core-compute-prod.internal
+        DM_STORE_APP_URL: http://dm-store-prod.service.core-compute-prod.internal
+    global:
```
Comment (luigibk) [inline k8s/ithc/common/em/em-ccd-orchestrator.yaml]: Same as above

File: k8s/ithc/common/em/em-npa.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  values:
+    npa:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/npa:prod-39c86832
+      ingressHost: em-npa-prod.service.core-compute-prod.internal
+      environment:
+        S2S_BASE_URI: "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal"
+        IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
+    global:
```
Comment (luigibk) [inline k8s/ithc/common/em/em-npa.yaml]: Same as above

File: k8s/ithc/common/em/em-stitching.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  values:
+    stitching:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/stitching:prod-4765e91f
+      ingressHost: em-stitching-prod.service.core-compute-prod.internal
+      environment:
+        DOCMOSIS_ENDPOINT: https://docmosis.platform.hmcts.net/rs/convert
+        IDAM_API_BASE_URI: https://idam-api.platform.hmcts.net
+        SPRING_DATASOURCE_URL: jdbc:postgresql://em-stitching-postgres-db-prod.postgres.database.azure.com:5432/emstitch?ssl=disable
```
Comment (luigibk) [inline k8s/ithc/common/em/em-stitching.yaml]: Same as above

File: k8s/ithc/common/em/identity.yaml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: em
+  namespace: em
+spec:
+  type: 0
+  ResourceID: "/subscriptions/8cbc6f36-7c56-4963-9d36-739db5d00b27/resourceGroups/managed-identities-prod-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/em-prod-mi"
+  ClientID: "87c2c28f-268c-437a-b346-0dd8bc5b16eb"
+
```
Comment (luigibk) [inline k8s/ithc/common/em/identity.yaml]: Same as above

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  rollback:
+    enable: true
+    force: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-ccd-orchestrator
+    version: 0.1.4
+  values:
+    ### EM-CCD-ORCHESTRATOR SERVICES ###
+    em-stitching:
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: Is this right ?

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+      java:
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: This is nested inside an alias

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+        releaseNameOverride: em-ccd-orchestrator-demo-stitching
+        environment:
+          SPRING_DATASOURCE_URL: 'jdbc:postgresql://em-ccd-orchestrator-demo-postgres-db:5432/postgres?ssl=disable'
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: Demo Databases won't be accessible here, they should be . You should use postgresql pod with persistence in Demo.

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+          S2S_BASE_URI: http://em-ccd-orchestrator-demo-s2s
+          DM_STORE_APP_URL: http://em-ccd-orchestrator-demo-dmstore
+          IDAM_API_BASE_URI: https://idam-api.demo.platform.hmcts.net
+
+    dm-store:
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: Is this a requirement in chart ? This won't like this way ?

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+      java:
+        releaseNameOverride: em-ccd-orchestrator-demo-dmstore
+        environment:
+          SPRING_DATASOURCE_URL: 'jdbc:postgresql://em-ccd-orchestrator-demo-postgres:5432/evidence'
+          IDAM_USER_BASE_URI: https://idam-api.demo.platform.hmcts.net
+          IDAM_S2S_BASE_URI: http://em-ccd-orchestrator-demo-s2s
+          ENABLE_DB_MIGRATE: 'true'
+          ENABLE_TTL: 'false'
+
+    ccd-case-management-web:
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: Is this a requirement in chart ? This won't like this way ?

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+      nodejs:
+        releaseNameOverride: em-ccd-orchestrator-demo-case-management-web
+        ingressHost: www-ccd-aks.demo.platform.hmcts.net
+
+    rpe-service-auth-provider:
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: Is this a requirement in chart ? This won't like this way ?

File: k8s/demo/common/em-ccd-orchestrator/namespace.yaml [hunk 14]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+   name: em-ccd-orchestrator
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-ccd-orchestrator/namespace.yaml]: Should be same namespace per team / product

File: k8s/demo/common/em-npa/em-npa-demo.yaml [hunk 15]
```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: flux.weave.works/v1beta1
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-npa/em-npa-demo.yaml]: Same comments as orchestrator, this wont work.

File: k8s/demo/common/em-stitching/em-stitching-demo.yaml [hunk 16]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (adusumillipraveen) [inline k8s/demo/common/em-stitching/em-stitching-demo.yaml]: same as orchestrator, this wont work

File: k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml (hunk 9 above)
Comment (tomxelliott) [reply k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml]: I have seen this done in most other charts in Demo -> see `k8s/demo/common/divorce/div-demo.yaml`and `k8s/demo/common/ccd/latest-ccd-chart.yaml`

**Pull request closed without merging**